### ISSUE TITLE
[REQ] Support useNullForUnknownEnumValue for python

### ISSUE THREAD

**Issue by slash-arun** [OpenAPITools/openapi-generator#4143]
### Is your feature request related to a problem? Please describe.

useNullForUnknownEnumValue is currently only supported for Java, it would be nice to have this for the python client as well

## Additional context

https://github.com/OpenAPITools/openapi-generator/issues/625

**Comment by spacether** [comment]
I don't think that returning a null value is a good solution here because some enums contain the null value (None).
A better solution may be to use a default unknown enum option. Someone wrote a I think global additional property that can be passed to our generate command.
